verilog/analysis/verilog_linter.cc

```diff
@@ -119,8 +119,8 @@ static void PrintFixAlternatives(std::ostream& stream, absl::string_view text,
   const bool print_alternative_number = fixes.size() > 1;
   for (size_t i = 0; i < fixes.size(); ++i) {
     if (print_alternative_number) {
-      stream << "=== " << (i + 1)
-             << ". alternative =================================\n";
+      stream << verible::term::inverse(
+          absl::StrCat("[ ", (i + 1), ". Alternative ]\n"));
```

Comment:
FYI: some autofixes have short descriptions (e.g. https://github.com/chipsalliance/verible/blob/master/verilog/analysis/checkers/endif_comment_rule.cc#L146 ), so maybe they could be used here.

Reply:
Good idea. Added the `Description()`.

Reply:
To make it maximally useful, in #942 I've added descriptions to all AutoFixes we have.